frontend/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../tsconfig-base",
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "noEmit": true,
+    "module": "esnext",
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true,
+    "paths": {
+      "@/*": ["./*"]
```

Comment:
It's a thing in TypeScript that seems to be an option in create-next-app now that aliases `@` to the root of the project yeah, so imagine you have `components/something/something-else/Thing.tsx`, and you have a util `util/thing.ts`. To import that util from the component, you'd have to do `import .. from '../../../util/thing'`, but with the `@` you can just do `import .. from '@/util/thing'` instead.

Doesn't really matter since hopefully everyone takes advantage of auto import features in their editor, but it doesn't hurt anything